### ISSUE TITLE
Hot reload

### ISSUE THREAD

**Issue by madnight** [y-taka-23/miso-tutorial-app#1]
Hi, thx for this awesome work. Is there any possibility for a live reload (recompile) on file change (*.hs)?

**Comment by y-taka-23** [comment]
Yes, but you have to renew the timestamp of `Main.hs` to recompile the artifacts. It's because that, unlike `elm-webpack-loader`, the `ghcjs-loader` module monitors only files which are directly referred from JavaScript.

**Comment by madnight** [comment]
Ah good to know, okay i'll try to find a workaround then to get this done
